pkgs/applications/misc/writefreely/default.nix

```
@@ -13,10 +13,19 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-CBPvtc3K9hr1oEmC+yUe3kPSWx20k6eMRqoxsf3NfCE=";
 
-  nativeBuildInputs = [ go-bindata ];
+  nativeBuildInputs = [ go-bindata lessc ];
 
   preBuild = ''
     make assets
+    export NODE_PATH=${nodePackages.less-plugin-clean-css}/lib/node_modules
+    cd less/
+    make all
+    cd ..
```

a better pattern for this would be to use subshells:
```suggestion
    (
      cd less
      make all
    )
```